docker/test/fasttest/Dockerfile

```diff
@@ -22,7 +22,7 @@ RUN apt-get update \
         zstd \
     --yes --no-install-recommends
 
-RUN pip3 install numpy scipy pandas Jinja2
+RUN pip3 install numpy scipy pandas==1.5.3 Jinja2
```

Comment:
Thank you! But we must pin all dependencies.
Let's at least pin everything in this line before merging.